cterasdk/edge/config.py

```diff
@@ -42,6 +45,15 @@ def set_hostname(self, hostname):
         logging.getLogger().info('Configuring device hostname. %s', {'hostname': hostname})
         return self._gateway.put('/config/device/hostname', hostname)
 
+    def export(self):
```

Comment:
I think you should add an optional file destination path

Reply:
Done